README.md

```diff
@@ -46,14 +46,35 @@
-small number of users.
-
-> Please note that this repository and its releases are a superset of the core repository.
+This is a repository for OpenTelemetry Collector components that are not suitable for the  [core repository](https://github.com/open-telemetry/opentelemetry-collector) of the collector. 
+
+The official distributions, core and contrib, are available as part of the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository. Some of the components in this repository are part of the "core" distribution, such as the Jaeger and Prometheus components, but most of the components here are only available as part of the "contrib" distribution. Users of the OpenTelemetry Collector are also encouraged to build their own custom distributions with the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder), using the components they need from the core repository, the contrib repository, and possibly third-party or internal repositories.
+
+The official distributions, core and contrib, are available as part of the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository.
+
+Each component has its own support levels, as defined in the following sections. For each signal that a component supports, there's a stability level, setting the right expectations. It is possible then that a component will be **Stable** for traces but **Alpha** for metrics and **In Development** for logs.
+
+## Stability levels
+
+While we intend to provide high-quality components as part of this repository, we acknowledge that not all of them are ready for prime time. As such, each component should list its current stability level for each telemetry signal, according to the following definitions:
+
+* **In development**: not all pieces of the component are in place yet and it might not be available as part of any distributions yet. Bugs and performance issues should be reported, but it is likely that the component owners might not give them much attention. Your feedback is still desired, especially when it comes to the user-experience (configuration options, component observability, technical implementation details, ...). Configuration options might break often depending on how things evolve. The component should not be used in production.
+* **Alpha**: the component is ready to be used for limited non-critical workloads and the authors of this component would welcome your feedback. Bugs and performance problems should be reported, but component owners might not work on them right away. The configuration options might change often without backwards compatibility guarantees.
+* **Beta**: same as Alpha, but the configuration options are deemed stable. While there might be breaking changes between releases, component owners should try to minimize them.
```

Comment:
Didn't you propose that **Beta** can be used in production while config might break? https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9412#discussion_r860038480

Reply:
I had only the config changes in mind based on previous comments, but we could say that people can have higher confidence in a beta component to run it with more workloads than they'd run with an alpha component. Let me draft something to that effect.

Reply:
> A component at this stage is expected to have had exposure to non-critical production workloads already during its **Alpha** phase, making it suitable for broader usage.

How about this?

Reply:
I think that wording is good.  I like keeping "Stable" as the only place to official say "this is officially GA".

Reply:
Although now I'm wondering if instead of "**Stable**: the component is ready to be used in production." we should say something like "**Stable**: the component is ready for general availability."  The form implies no stage before "Stable" should ever be used in production, but there are times Beta could be used.  The later implies this component has had a "production" release, thus making it safe to use.

Reply:
Makes sense to me

Reply:
PR updated.